src/EventSubscriber/CentreonEventSubscriber.php

```diff
@@ -306,10 +309,16 @@ public function onKernelException(ExceptionEvent $event)
          * we create a custom error message.
          * If we don't do that a HTML error will appeared.
          */
-        if ($errorIsBeforeController && $event->getException()->getCode() !== 403) {
-            $errorCode = $event->getException()->getCode() > 0
-                ? $event->getException()->getCode()
-                : Response::HTTP_INTERNAL_SERVER_ERROR;
+        if ($errorIsBeforeController) {
+            if ($event->getException()->getCode() !== 403) {
+                $errorCode = $event->getException()->getCode() > 0
+                    ? $event->getException()->getCode()
+                    : Response::HTTP_INTERNAL_SERVER_ERROR;
+                $statusCode = Response::HTTP_INTERNAL_SERVER_ERROR;
+            } else {
+                $errorCode = $event->getException()->getCode();
+                $statusCode = Response::HTTP_FORBIDDEN;
```

Comment:
unauthorized

Reply:
according to rfc, it is 403 :) https://tools.ietf.org/html/rfc7231#section-6.5.3